azavea/cac-tripplanner
# Switch all IPs to the new VirtualBox default range

## Overview

This should allow spinning up the project without creating a `networks.conf` file.


### Demo

N/A

### Notes

N/A

## Testing Instructions

 * If you've spun up the project before, remove `/etc/vbox/networks.conf`.
 * If you've spun up this project before, edit `deployment/ansible/group_vars/development` (which is `.gitignore`d) and replace all instances of `192.168.8` with `192.168.56` (that is, the address `192.168.8.26` should become `192.168.56.26`)
 * If you already have a `Graph.obj` in `otp_data/` then run `vagrant up` and then `vagrant provision database app`.
 * If you don't have an existing Graph.obj in `otp_data/` you'll need to follow the deployment instructions for downloading data and building a new graph. Then run `vagrant up` and `vagrant provision`.
 * Launch the app locally and confirm that it works (including routing ~--my graph hasn't finished building yet so I am putting this PR up optimistically~).


## Checklist


Closes #1302
